src/librustc/session/mod.rs

```diff
@@ -64,12 +64,15 @@ pub struct Session {
     pub target: config::Config,
     pub host: Target,
     pub opts: config::Options,
+    pub host_tlib_path: SearchPath,
+    /// This is `None` if the host and target are the same.
+    pub target_tlib_path: Option<SearchPath>,
     pub parse_sess: ParseSess,
     /// For a library crate, this is always none
     pub entry_fn: Once<Option<(NodeId, Span, config::EntryFnType)>>,
     pub plugin_registrar_fn: Once<Option<ast::NodeId>>,
     pub derive_registrar_fn: Once<Option<ast::NodeId>>,
-    pub default_sysroot: Option<PathBuf>,
+    pub sysroot: PathBuf,
```

Comment:
cc @oli-obk @Manishearth Do miri and/or clippy need `default_sysroot` at all?

Reply:
a quick grep into these two tools showed no uses

Reply:
Oh wait I can just `git submodule update --recursive --init; rg default_sysroot`.

This field wasn't even publicly used, *anywhere*.